<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBVaccinationProbeEditor</class>
 <widget class="QDialog" name="RBVaccinationProbeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMainInfo">
      <attribute name="title">
       <string>Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblAge">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Возраст</string>
         </property>
         <property name="buddy">
          <cstring>cmbBegAgeUnit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QComboBox" name="cmbBegAgeUnit">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Д</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Н</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>М</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Г</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtBegAgeCount">
           <property name="inputMask">
            <string>000; </string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblAgeSep">
           <property name="text">
            <string>по</string>
           </property>
           <property name="buddy">
            <cstring>cmbEndAgeUnit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbEndAgeUnit">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Д</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Н</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>М</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Г</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtEndAgeCount">
           <property name="inputMask">
            <string>000; </string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblDuration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Срок действия</string>
         </property>
         <property name="buddy">
          <cstring>cmbDurationUnit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblMinimumTerm">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Минимальный срок</string>
         </property>
         <property name="buddy">
          <cstring>cmbMinimumTermUnit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="cmbMinimumTermUnit">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Д</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Н</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>М</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Г</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtMinimumTermCount">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="cmbDurationUnit">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Д</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Н</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>М</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Г</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtDurationCount">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblInfections">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Инфекции</string>
         </property>
         <property name="buddy">
          <cstring>cmbInfections</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="CRBMultivalueComboBox" name="cmbInfections"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblVaccinationType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Тип прививки</string>
         </property>
         <property name="buddy">
          <cstring>edtVaccinationType</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edtVaccinationType">
         <property name="maxLength">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBMultivalueComboBox</class>
   <extends>QComboBox</extends>
   <header>library/MultivalueComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbBegAgeUnit</tabstop>
  <tabstop>edtBegAgeCount</tabstop>
  <tabstop>cmbEndAgeUnit</tabstop>
  <tabstop>edtEndAgeCount</tabstop>
  <tabstop>cmbMinimumTermUnit</tabstop>
  <tabstop>edtMinimumTermCount</tabstop>
  <tabstop>cmbDurationUnit</tabstop>
  <tabstop>edtDurationCount</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RBVaccinationProbeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBVaccinationProbeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
